blockFromPersonalCalendar.gs

```diff
@@ -10,33 +15,41 @@
  * - Set a trigger for an hourly run of `blockFromPersonalCalendar` 
  */
 const CONFIG = {
-  calendarId: "marianosimone@gmail.com", // (personal) calendar from which to block time
+  calendarIds: ["mattkharris@gmail.com", "aqv06aedps4ek5pggj5ug1gots@group.calendar.google.com"], // (personal) calendars from which to block time
```

Comment:
would you mind changing this to `first-calendar@gmail.com` and `second-calendar@gmail.com` or something similar? I initially put my email there, but now I realize that this is intended for copy&pasting by other people, and making it explicit would be great.